confluent_kafka/src/Consumer.c

```diff
@@ -341,7 +341,7 @@ static PyObject *Consumer_store_offsets (Handle *self, PyObject *args,
 		     "buildtime 0x%x)",
 		     rd_kafka_version(), RD_KAFKA_VERSION);
 	return NULL;
-#endif
```

Comment:
With this #else guard the variables defined above will be unused and produce warnings.
I suggest moving the variable decls down to the #else block.